.github/workflows/commit-lint.yml

```diff
@@ -0,0 +1,25 @@
+name: Semantic release
+on: [pull_request]
+
+jobs:
+  semantic-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Dry-run release
+        uses: tradeshift/actions-semantic-release@v1
+        id: semantic-release
+        with:
+          branches: |
+            [
+              'master'
+            ]
```

Comment:
We actually don't need this field any more, this should be the default. I'm removing it where I remember :grimacing: